packages/templates/src/utils/memoize.ts

```diff
@@ -13,9 +13,14 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-export default function memoize(func, argsHash = (args) => args.join('-')) {
+
+type ArgHash = (args: string[] | number[]) => string;
+export default function memoize<T>(
```

Comment:
BTW, why does this function exist in this package? Don't we already have a global one or at least we can import one? Or is it specific to what we need it to do in this package?

Reply:
As far as I know, we don't have a global one.